zubbi/templates/search.html

```diff
@@ -92,7 +92,7 @@ <h5 class="card-title">
           {{ match.name }}
         </h5>
         <h6 class="card-subtitle mb-2 text-muted">{{ match.repo }}</h6>
-        <p class="card-text">{% if match.description %}{{ match.description.split('\n')|first }}{% endif %}</p>
+        <p class="card-text">{% if match.description %}{{ match.description_rendered.split('\n')|first|striptags }}{% endif %}</p>
```

Comment:
I like the idea but I think we should also adapt the check if the attribute exists and if `description_rendered` is not set, we might fall back to the description 😉